build.gradle

```diff
@@ -77,7 +77,7 @@ jacocoTestCoverageVerification {
     afterEvaluate {
         classDirectories.setFrom(files(classDirectories.files.collect {
             fileTree(dir: it, exclude:
```

Comment:
tu możesz dodać jeszcze taką samą regułę dla raportu - inaczej raport będzie pokazywał co innego niż samo sprawdzenie:
jacocoTestReport {
    afterEvaluate {
        classDirectories.setFrom(files(classDirectories.files.collect {
            fileTree(dir: it, exclude:
                    ['pl/futurecollars/invoicing/App.class',
                     'pl/futurecollars/invoicing/controller/**'
                    ])
        }))
    }
}